### PR TITLE
[RAID] Naxxramas - Loatheb bugfixes

### PR DESCRIPTION
**Pull request by Pondaveia**
**Thank-you FALL1N1 for the [fix](https://github.com/azerothcore/azerothcore-wotlk/pull/936)**

# Loatheb Bugfixes

##### CHANGES PROPOSED:
- All left spores will now despawn upon boss death or reset.
- Fixed Necrotic Aura's timer.
- New method of spawning spores, we don't have to share them with deathbloom as it has different timer.
- Boss now evades if pulled outside the combat area.

##### TESTS PERFORMED:

- Compiled and tested

##### HOW TO TEST THE CHANGES:

(Make sure you're in god mode before the first command or be ready to engage combat)
1. .go c 128066

##### Target branch(es):

Master

### CODE REVIEW

**Comment by poszer** [comment]
Could you fix errors @Pondaveia  :smiley:

File: src/server/scripts/Northrend/Naxxramas/boss_loatheb.cpp [hunk 1]
```diff
@@ -44,58 +45,76 @@ class boss_loatheb : public CreatureScript
...
+        boss_loathebAI(Creature *c) : BossAI(c, BOSS_LOATHEB), summons(me)
         {
             pInstance = me->GetInstanceScript();
+            me->SetHomePosition(me->GetPositionX(), me->GetPositionY(), me->GetPositionZ(), me->GetOrientation());
         }
 
         InstanceScript* pInstance;
         EventMap events;
+        SummonList summons;
+        Position p = me->GetHomePosition();
```
Comment (Nefertumm) [inline src/server/scripts/Northrend/Naxxramas/boss_loatheb.cpp]: Why? You never use it :P

Comment (Pondaveia) [reply src/server/scripts/Northrend/Naxxramas/boss_loatheb.cpp]: Forgot to delete :thinking:

**Comment by Knindzagxg** [comment]
Tested and works very nice.

Tho, one mhm, weird thing happened as well.
I got Undying achie once i killed him.

It was single boss test, and haven't killed any before, which u can see in image as well "You are now saved to this instance."


https://i.gyazo.com/5a6c1afacf27033b92d202402d53104a.png

**Comment by Pondaveia** [comment]
@Knindzagxg This is another bug though, a check on instance_naxxramas. Can you report it here? #1657 Thanks for the test 👍

**Comment by Pondaveia** [comment]
#1657 This PR will be included in this Overhaul.

**Pull request closed without merging**